content/en/docs/v3.5/install.md

```diff
@@ -9,86 +9,160 @@ minGoVers: 1.20
 
 Before installing etcd, see the following pages:
 
-- [Supported platforms][]
-- [Hardware recommendations][]
+- [Supported platforms](/docs/v3.5/op-guide/supported-platform)
+- [Hardware recommendations](/docs/v3.5/op-guide/hardware)
 
 ## Install pre-built binaries
 
-The easiest way to install etcd is from pre-built binaries:
+Install `etcd` from pre-built binaries using cURL.
```

Comment:
I would remove `using cURL` explicitly as other tools can be used. I like the steps you provided below.